src/Compilers/VisualBasic/Portable/PublicAPI.Unshipped.txt

```diff
@@ -1,2 +1,3 @@
 *REMOVED*Overrides Microsoft.CodeAnalysis.VisualBasic.VisualBasicCompilationOptions.GetHashCode() -> Integer
 Microsoft.CodeAnalysis.VisualBasicExtensions.ContainsDirective(node As Microsoft.CodeAnalysis.SyntaxNode, kind As Microsoft.CodeAnalysis.VisualBasic.SyntaxKind) -> Boolean
+Microsoft.CodeAnalysis.VisualBasic.SyntaxFacts.IsIdentifierContainerDirectiveTrivia(token As Microsoft.CodeAnalysis.VisualBasic.SyntaxKind) -> Boolean
```

Comment:
unclear why we need these new methods on the public surface of the compiler.

Reply:
There is no real reason other than that's the place where I found they could be nicely grouped with the others, as this serves as a handy extension shared across multiple points in the work for pp symbols.